go.mod

```diff
@@ -1,16 +1,15 @@
 module mvdan.cc/gibot
 
 require (
-	github.com/go-kit/kit v0.8.0 // indirect
-	github.com/go-logfmt/logfmt v0.4.0 // indirect
-	github.com/go-stack/stack v1.8.0 // indirect
+	github.com/go-kit/kit v0.9.0 // indirect
 	github.com/google/go-github v17.0.0+incompatible
-	github.com/nickvanw/ircx v0.0.0-20181210092626-1a8800ff55e7
-	github.com/sorcix/irc v1.1.4
-	github.com/xanzy/go-gitlab v0.12.2
-	golang.org/x/net v0.0.0-20181220203305-927f97764cc3 // indirect
-	golang.org/x/oauth2 v0.0.0-20181203162652-d668ce993890 // indirect
-	golang.org/x/sync v0.0.0-20181221193216-37e7f081c4d4 // indirect
-	google.golang.org/appengine v1.4.0 // indirect
+	github.com/nickvanw/ircx/v2 v2.0.0
+	github.com/xanzy/go-gitlab v0.18.0
+	golang.org/x/net v0.0.0-20190628185345-da137c7871d7 // indirect
+	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45 // indirect
+	google.golang.org/appengine v1.6.1 // indirect
+	gopkg.in/sorcix/irc.v2 v2.0.0-20180626144439-63eed78b082d
 	mvdan.cc/xurls/v2 v2.0.0
 )
+
+go 1.13
```

Comment:
Shouldn’t this state go 1.12?

Reply:
I initially thought this was a "base" or minimum version, but it turns out it's more like a "target" version. So I now generally let the `go` tool of the version I'm using to develop to write the `go X` line. I develop on master, so that's 1.13 at the moment.